members-only/config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+
+  get  '/signup',  to: 'users#new'
```

Comment:
There's no `new` action in the user controller, hence I can't signup as a new user.

![image](https://user-images.githubusercontent.com/36057474/70844550-26c04b80-1e43-11ea-996b-03e141befee6.png)